docs/source/overlay_design_methodology/verilog_overlay_tutorial.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
I think this documentation should be carefully reviewed for formatting and wording. We should rephrase sentences like "I developed this demo on Vivado 2019.2 using a TUL-2 board. I am still learning this specific tool myself so there may be improvements". I also noticed a lot of formatting issues.

Some pictures are not rendering. I think we need to use a consistent naming convention in the image folder - they all use small letters so please adjust this for "axiDesignSource.png" and "ZynqAndSimpleLogicModule.png"